packages/buildpacks/test/mocha.opts

```diff
@@ -4,4 +4,4 @@
 --watch-extensions ts
 --recursive
 --reporter spec
---timeout 100000
+--timeout 50000
```

Comment:
This seems to be decreasing the timeout?

Reply:
This was 1000 seconds (an error?) so I dropped it in-line with the rest.